src/main/java/fr/alasdiablo/janoeo/init/OverworldOresBlocks.java

```diff
@@ -3,94 +3,76 @@
-    /**
-     * Instance of Overworld Lead Ore Block
-     */
-    public static Block LEAD_ORE
-            = new BasicOre(Registries.LEAD_ORE, 1, NULL);
-    /**
-     * Instance of Overworld Ruby Ore Block
-     */
-    public static Block RUBY_ORE
-            = new BasicOre(Registries.RUBY_ORE, 2, RARE);
-    /**
-     * Instance of Overworld Sapphire Ore Block
-     */
-    public static Block SAPPHIRE_ORE
-            = new BasicOre(Registries.SAPPHIRE_ORE, 2, RARE);
-    /**
-     * Instance of Overworld Amethyst Ore Block
-     */
-    public static Block AMETHYST_ORE
-            = new BasicOre(Registries.AMETHYST_ORE, 2, RARE);
-    /**
-     * Instance of Overworld Zinc Ore Block
-     */
-    public static Block ZINC_ORE
-            = new BasicOre(Registries.ZINC_ORE, 1, NULL);
-    /**
-     * Instance of Overworld Osmium Ore Block
-     */
-    public static Block OSMIUM_ORE
-            = new BasicOre(Registries.OSMIUM_ORE, 2, NULL);
+
+    public static HashMap<String, Block> Ores = new HashMap<String, Block>() {{
+        // --------------------------------------- overworld.stone --------------------------------------
+        // --> MODDED ORE
+        put("ALUMINIUM_ORE",       new BasicOre(Registries.ALUMINIUM_ORE,  	1, NULL));
+        put("AMBER_ORE",           new BasicOre(Registries.AMBER_ORE,  	    1, NULL));
+        put("AMETHYST_ORE",        new BasicOre(Registries.AMETHYST_ORE,    1, NULL));
+        put("APATITE_ORE",         new BasicOre(Registries.APATITE_ORE,    	1, NULL));
+        put("BITUMINOUS_ORE",      new BasicOre(Registries.BITUMINOUS_ORE,  1, NULL));
+        put("CINNABAR_ORE",        new BasicOre(Registries.CINNABAR_ORE,   	1, NULL));
+        put("COPPER_ORE",          new BasicOre(Registries.COPPER_ORE,      1, NULL));
+        put("LEAD_ORE",            new BasicOre(Registries.LEAD_ORE,   	    1, NULL));
+        put("NICKEL_ORE",          new BasicOre(Registries.NICKEL_ORE,     	1, NULL));
+        put("NITER_ORE",           new BasicOre(Registries.NITER_ORE,      	1, NULL));
+        put("OSMIUM_ORE",          new BasicOre(Registries.OSMIUM_ORE,     	1, NULL));
+        put("PLATINUM_ORE",        new BasicOre(Registries.PLATINUM_ORE,   	1, NULL));
+        put("RUBY_ORE",            new BasicOre(Registries.RUBY_ORE,       	1, NULL));
+        put("SALTPETER_ORE",       new BasicOre(Registries.SALTPETER_ORE,  	1, NULL));
+        put("SAPPHIRE_ORE",        new BasicOre(Registries.SAPPHIRE_ORE,   	1, NULL));
+        put("SILVER_ORE",          new BasicOre(Registries.SILVER_ORE, 	    1, NULL));
+        put("SULFUR_ORE",          new BasicOre(Registries.SULFUR_ORE,     	1, NULL));
+        put("TIN_ORE",             new BasicOre(Registries.TIN_ORE,        	1, NULL));
+        put("URANIUM_ORE",         new BasicOre(Registries.URANIUM_ORE,    	1, NULL));
+        put("ZINC_ORE",            new BasicOre(Registries.ZINC_ORE,       	1, NULL));
+    }};
```

Comment:
+ The Map can be final.
+ Do not use HashMap as base variable type (use Map instead)
+ The key of this Map, make the mod less modular
+ `new HashMap<String, Block>()` can be replaced by `new HashMap<>()` we already know the type
+ Variable name like a class ?